Dockerfile

```
@@ -1,4 +1,4 @@
-FROM golang:1.20-alpine3.17 as builder
+FROM golang:alpine3.19 as builder
```

The above will always fetch the latest major version of Go which might lead to unexpected failures. Let's use something like below:
```suggestion
FROM golang:1.22-alpine3.19 as builder
```